omero/sysadmins/server-upgrade.rst

```diff
@@ -46,6 +46,20 @@ to enable successful upgrading of the database, otherwise the upgrade
 script aborts with a message saying that your database server version is
 less than the OMERO prerequisite.
 
+Corrupted pyramids
+^^^^^^^^^^^^^^^^^^
+
+A bug introduced in OMERO 5.2 meant that corrupted pyramids were generated for
+large TIFF files, ignoring the little endian encoding. This bug was fixed in
+OMERO 5.4.4 and corrupted pyramids need to be deleted to allow new ones to be
+generated::
+
+   bin/omero admin removepyramids
```

Comment:
--little-endian should be specified since it affects mainly little-endian=true pyramid